api/ereqs_admin/static/admin/module/_editor.scss

```diff
@@ -6,6 +6,17 @@
     background: lightpink;
   }
 
+  .inline-footnote {
+    color: $color-pink;
+    @include font-san-serif;
+  }
+
+  .inline-footnote::before {
+    content: 'Footnote ' attr(data-emblem) ': ';
+    padding-left: 0.5em;
+    @include font-san-serif-bold;
```

Comment:
Uh, is there a particular reason we're calling the font `san-serif` instead of `sans-serif`?

Reply:
Good question; we can probably swap them all to sans. Let's check in with design real quick to confirm we want to change the font face in the middle of a paragraph, though.

Reply:
Oh, I styled it this way b/c it's what Austin's mockups looked like, but yeah, we should double-check.